### PR TITLE
docs(esl-share): update README

### PR DESCRIPTION
**Pull request by dshovchko**

### CODE REVIEW

**Review by ala-n — APPROVED** [review]

**Review by NastaLeo — APPROVED** [review]

**Review by julia-murashko — APPROVED** [review]

File: src/modules/esl-share/README.md [hunk 1]
```diff
@@ -272,6 +274,30 @@ The principle of cascading is similar to CSS variables. The value is searched fr
...
+ - `groups` - getter that returns list of all available groups
+ - `get` - selects the buttons for the given list and returns their configuration
+ - `clear` - clears the configuration
+ - `getButton` - gets the button configuration for the given name
+ - `getGroup` - gets the group of buttons configuration for the given name
+ - `append` - updates the configuration with a `ESLShareButtonConfig`(s) or `ESLShareGroupConfig`(s)
+
+#### Events
+
+ - `change` - event to dispatch on change of ESLShareConfig
```
Comment (fshovchko) [inline src/modules/esl-share/README.md:299]:
```suggestion
 - `change` - event to dispatch on change of `ESLShareConfig`
```

File: src/modules/esl-share/README.md [hunk 2]
```diff
@@ -272,6 +274,30 @@ The principle of cascading is similar to CSS variables. The value is searched fr
...
+#### Events
+
+ - `esl:share:changed` - event to dispatch on change of ESLShareButton
+ - `esl:share:ready` - event to dispatch on ready of ESLShareButton
+
+### ESLShareConfig
+
+#### Public API
+
+ - `instance` - static getter that returns shared instance of ESLShareConfig
```
Comment (fshovchko) [inline src/modules/esl-share/README.md:286]:
```suggestion
 - `instance` - static getter that returns shared instance of `ESLShareConfig`
```

File: src/modules/esl-share/README.md [hunk 3]
```diff
@@ -306,43 +333,52 @@ The popup is notified of any configuration changes. In this case, the component
...
-
-There are two modes available to render buttons. In list mode, the `ESLShareList` is drawn inside the component as a list. Nothing special. 
-
-Similarly, in popup mode, the `ESLSharePopupTrigger`` is drawn inside component and shows a popup with the specified list of buttons.
-
-#### Public API
+#### Events
 
- - `config` - static method to get or update config with a promise of a new config object or using a config provider function (***Deprecated***)
+ - `esl:share:ready` - event to dispatch on ready of ESLShare
```
Comment (fshovchko) [inline src/modules/esl-share/README.md:351]:
```suggestion
 - `esl:share:ready` - event to dispatch on ready of `ESLShare`
```

**Review by HenadzV — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Review by Natalie-Smirnova — APPROVED** [review]

File: src/modules/esl-share/README.md [hunk 4]
```diff
@@ -238,12 +229,22 @@ ESLShare provides several actions available for you to use:
...
 For using actions you should import the required actions before setting up the configuration and registering ESLShare components. When an unregistered action is specified for a button, the button will not be able to perform the share action and will be marked as 'unavailable'. The same behavior occurs if the action is unavailable on the user's device, e.g. native action on the user's desktop.
 
-Note: if default configuration does not used, you should import the required actions manually:
+Note: if the default configuration does not used, you should import the required actions manually:
 ```typescript
   import '@exadel/esl/modules/esl-share/actions/copy';
   import '@exadel/esl/modules/esl-share/actions/media';
 ```
 
+### List of buttons through which you can share
```
[the remark this answers is not in the document]
Comment (ala-n) [reply src/modules/esl-share/README.md:238]: Just one more idea
```suggestion
### List of OOTB ways of sharing
```

File: src/modules/esl-share/README.md [hunk 5]
```diff
@@ -306,43 +333,52 @@ The popup is notified of any configuration changes. In this case, the component
...
+ - `myspace` - shares data into MySpace a social media platform
+ - `native-share` - activates the native sharing mechanism of the device to share data
+ - `pinterest` - shares data into Pinterest a social curation website for sharing and categorizing images found online
+ - `print` - prints page
+ - `pusha` - shares data into Pusha a social network service
+ - `reddit` - shares data into Reddit an American social news aggregation, content rating, and discussion website 
+ - `sina-weibo` - shares data into Sina Weibo a microblogging website and app which compares to Twitter and Instagram
+ - `telegram` - shares data into Telegram a cloud-based, cross-platform instant messaging (IM) service
+ - `twitter` - shares data into Twitter a free social networking site where users broadcast short posts known as tweets
+ - `wykop` - shares data into Wykop a Polish social networking internet service
```
Comment (ala-n) [inline src/modules/esl-share/README.md:384]: Just a cosmetic suggestion:
possibly it makes sense to split into two lists social media and special actions (copy, print, native, mail)

```suggestion
 - Social medias:
    - `blogger`- shares data into Blogger an online content management system
    - `facebook` - shares data into Facebook a free social networking website
    - `hatena` - shares data into Hatena a japan free blogging service
    - `kakao` - shares data into Kakao the most popular network among Koreans
    - `line` - shares data into Line a popular messaging and social media platform in Japan and other parts of Asia
    - `linkedin` - shares data into LinkedIn a social networking site designed specifically for the business community
    - `mix` - shares data into Mix a social network service
    - `mixi` - shares data into Mixi an online Japanese social networking service
    - `myspace` - shares data into MySpace a social media platform
    - `pinterest` - shares data into Pinterest a social curation website for sharing and categorizing images found online
    - `pusha` - shares data into Pusha a social network service
    - `reddit` - shares data into Reddit an American social news aggregation, content rating, and discussion website 
    - `sina-weibo` - shares data into Sina Weibo a microblogging website and app which compares to Twitter and Instagram
    - `telegram` - shares data into Telegram a cloud-based, cross-platform instant messaging (IM) service
    - `twitter` - shares data into Twitter a free social networking site where users broadcast short posts known as tweets
    - `wykop` - shares data into Wykop a Polish social networking internet service
 - Simple Actions
   - `copy` - copies the share data into a clipboard
   - `mail` - activates the default mail client on the device for sending data via e-mail
   - `native-share` - activates the native sharing mechanism of the device to share data
   - `print` - prints page
```

**Review by abarmina — APPROVED** [review]

**Pull request merged**